.github/workflows/testing.yml

```diff
@@ -31,6 +31,15 @@ jobs:
                       php: 7.4
                     - laravel: 9.*
                       phpunit: 8.*
+                    - laravel: 10.*
+                      php: 7.3
+                    - laravel: 10.*
+                      php: 7.4
+                    - laravel: 10.*
+                      php: 8.0
+                    - laravel: 10.*
+                      phpunit: 8.*
+        
```

Comment:
I think it's time to NOT exclude the L10 tests, to make sure they are green. Right?

Reply:
I think those should actually be excluded; Laravel 10 only works with PHP >=8.1 (and PHPUnit 9 or 10), so running tests on lower PHP versions will immediately fail on the `composer install` operation.